kubernetes/linera-validator/redeploy.sh

```diff
@@ -9,3 +9,10 @@ echo "Pods:";
 kubectl get pods;
 echo -e "\nServices:";
 kubectl get svc;
+sleep 2;
+kubectl exec -it shards-0  -- cat wallet.json > /tmp/wallet.json;
```

Comment:
Idk why, but for some reason this is not copying the entire `json` file for me, making it invalid 🤔
I wonder if there's some limitation on the number of lines for cat or something weird like that, but there shouldn't be

Reply:
I've fixed this on #1016, by using `docker cp`. Feel free to copy those changes to this PR :)